mavros_extras/src/plugins/rpm.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * @brief RPM plugin
+ * @file rpm.cpp
+ * @author Pavlo Kolomiiets <pkolomiets@gmail.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2017 Pavlo Kolomiiets.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+#include <mavros_msgs/RPM.h>
+
+namespace mavros {
+namespace extra_plugins {
+/**
+ * @brief Ardupilot rpm plugin.
+ *
+ * This plugin allows publishing RPM sensor data from Ardupilot FCU to ROS.
+ *
+ */
+class RPMPlugin : public plugin::PluginBase {
+public:
+	RPMPlugin() : PluginBase(),
+		nh("~")
+	{ }
+
+	void initialize(UAS &uas_)
+	{
+		PluginBase::initialize(uas_);
+
+		rpm_pub = nh.advertise<mavros_msgs::RPM>("rpm", 10);
+	}
+
+	Subscriptions get_subscriptions()
+	{
+		return {
+			make_handler(&RPMPlugin::handle_rpm)
+		};
+	}
+
+private:
+	ros::NodeHandle nh;
+
+	ros::Publisher rpm_pub;
+
```

Comment:
Can you please add `Doxygen` like docs this as in `imu.cpp`? Ex: https://github.com/mavlink/mavros/blob/master/mavros/src/plugins/imu.cpp#L254-L259.
Thanks